recipes-support/lvrt-logging/lvrt-logging.bb

```diff
@@ -0,0 +1,11 @@
+SUMMARY = "INI file to specify logging paths for LVRT"
```

Comment:
INI file is a implementation detail that doesn't make sense to go in package summary. Just say `Configuration for NILRT Logging` or such.

Reply:
I have fixed this now.